src/styles/variables.scss

```diff
@@ -0,0 +1,8 @@
+$card-width: 200px;
+
+%display-setting {
+  display: grid;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
should this be inside `variables` file?

Reply:
I am not sure now...this is a placeholder. Should I keep it in catalog.scss?